results.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/python3
+import re
+
+# Part 1
+# The first part checks how many vowels there are in each
+# string. Therefore, it checks every letter and when it finds
+# a vowels it increments the counter. 
+# The second property is found using a regex, which checks
+# if there is at least a pair of the same letter. The last property
+# is found by simply checking if there are the unwanted srings.
+
+#fone = open("output_1", "w")
+
+vowels = ["a", "e", "i", "o", "u"]
+
+firstCounter = 0
+with open("input") as in_file:
+	for row in in_file:
+		count_vowels = 0
+		for r in row:
```

Comment:
Since you're solving the problem rule by rule, you would highly improve the readability of the solution if you split the logic into 3 methods (at least). One per rule you're testing.
Also, the performance could be improved because you could "fail fast", meaning that if you find one of the rules breaking, then you wouldn't have to test for the others.